packages/ui/src/components/va-select/VaSelect.vue

```diff
@@ -250,6 +250,7 @@ export default defineComponent({
     bordered: { type: Boolean, default: false },
     label: { type: String, default: '' },
     placeholder: { type: String, default: '' },
+    searchtext: { type: String, default: 'Search' }, //placeholder for search bar
```

Comment:
Maybe `searchPlaceholder` is better
```suggestion
    searchPlaceholder: { type: String, default: 'Search' }, //placeholder for search bar
```